docs/markdown/Dependencies.md

```diff
@@ -186,6 +186,24 @@ have been compiled for single-threaded use instead.
 
 `method` may be `auto`, `config-tool`, `pkg-config` or `extraframework`.
 
+## Dub
```

Comment:
This is added in the wrong place. This is in the middle of a list of names handed to `dependency()` which get special treatment.
A separate section is probably needed to discuss values for the `method:` keyword of `dependency()`.